walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -91,42 +92,43 @@
 
   /**
    * Recursive helper function used to find all combinations of size r of the input array arr[].
-   * chosen[]: temporary array to store current combination.
-   * start & end: staring and ending indexes in arr[].
-   * index: current index in data[].
+   * @param arr[] The input array.
+   * @param chosen[] Temporary array to store current combination.
+   * @param start Starting index in arr[].
+   * @param end Ending index in arr[].
+   * @param index Current index in chosen[].
```

Comment:
A little more elaborate names and explanations as to what each one is used for. Its not obvious on first glance.

For all the params.

Reply:
So this is a standard algorithm for finding combinations, and I really can't think of better descriptions for those parameters...

Reply:
To be honest, I can't say that I understand this algorithm 100%.